build_node/utils/git_sources_utils.py

```diff
@@ -95,3 +95,19 @@ def download_all(self):
         if code != 0:
             return False
         return True
+
+
+class FedpkgDownloader(BaseSourceDownloader):
+
+    def download_source(self, checksum: str, dst_path: str) -> str:
+        pass
+
+    def download_all(self):
+        sources_file = os.path.join(self._sources_dir, 'sources')
+        if not os.path.isfile(sources_file):
+            return False
+        code, out, err = local['fedpkg'].with_cwd(self._sources_dir).run(
+            args=('sources', '--force'), retcode=None)
```

Comment:
`--force` does not apply to `fedpkg sources` command (check the [manpage](https://linux.die.net/man/1/fedpkg)), so I'd say that you can safely remove it.